### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/kubeflow/website/issues/2656

### CODE REVIEW

**Comment by Arhell** [comment]
relative fixes
https://github.com/kubeflow/website/pull/2652 (same)

**Comment by Arhell** [comment]
cc @RFMVasconcelos

File: content/en/docs/reference/pytorchjob/v1/pytorch.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ string
 <td>
 <code>metadata</code></br>
 <em>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#objectmeta-v1-meta">
+<a href="https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta">
```
Comment (andreyvelich) [inline content/en/docs/reference/pytorchjob/v1/pytorch.md:59]: Please can you also fix the link for the MPIJob: https://www.kubeflow.org/docs/reference/mpijob/v1alpha2/mpi/#github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v1alpha2.MPIJob

**Review by andreyvelich — COMMENTED** [review]
Thank you for this fix @Arhell!
One small nit from my side.

File: content/en/docs/reference/pytorchjob/v1/pytorch.md (hunk 1 above)
Comment (Arhell) [reply content/en/docs/reference/pytorchjob/v1/pytorch.md:59]: @andreyvelich done

**Comment by RFMVasconcelos** [comment]
Thank you @Arhell and @andreyvelich! 
/lgtm

**Review by RFMVasconcelos — COMMENTED** [review]
Looks like we're good to go!

/approve

**Pull request merged**